.gitignore

```diff
@@ -289,8 +289,8 @@ __pycache__/
 
 # Virtual environment
 env/
-env27/
```

Comment:
has python 2 not been removed?

Reply:
Not in our .gitignore, at least - I can remove the newly added line if we want, just figured maybe there's someone out there with a 2.x env folder floating around :)